datalad_next/patches/tests/test_run.py

```diff
@@ -23,3 +26,79 @@ def test_substitution_config_default(existing_dataset):
     # make sure we could actually detect breakage with the check above
     with pytest.raises(IncompleteResultsError):
         ds.run('{python} -c "breakage"', result_renderer='disabled')
+
+
+def test_runrecord_portable_paths(existing_dataset):
+    ds = existing_dataset
+    dsrepo = ds.repo
+    infile = ds.pathobj / 'inputs' / 'testfile.txt'
+    outfile = ds.pathobj / 'outputs' / 'testfile.txt'
+    infile.parent.mkdir()
+    outfile.parent.mkdir()
+    infile.touch()
+    ds.save()
+    assert not outfile.exists()
+    # script copies any 'inputs' to the outputs dir
+    res = ds.run(
+        '{python} -c "'
+        'from shutil import copyfile;'
+        'from pathlib import Path;'
+        r'[copyfile(f, Path.cwd() / \'outputs\' / Path(f).name)'
```

Comment:
```suggestion
        r'[copyfile(f, Path.cwd() / \"outputs\" / Path(f).name)'
```

There is one current test failure because of this
```
[INFO] == Command start (output follows) ===== 
  File "<string>", line 1
    from shutil import copyfile;from pathlib import Path;[copyfile(f, Path.cwd() / \'outputs\' / Path(f).name)  for f in "inputs/testfile.txt".split()]
                                                                                                                                                      ^
SyntaxError: unexpected character after line continuation character
[INFO] == Command exit (modification check follows) ===== 
FAILED
```

Reply:
I actually changed it from the alternative you propose -- but that fails on windows due to an unfortunate quoting behavior